### PR TITLE
Install bundler before running bundle install

### PR DESCRIPTION
**Pull request by lpusok**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -117,49 +117,10 @@ func findMostRootPodfile(dir string) (string, error) {
...
-		return "", err
-	}
-	return cocoapodsVersionFromGemfileLockContent(content), nil
-}
-
 func cocoapodsVersionFromPodfileLockContent(content string) string {
 	exp := regexp.MustCompile("COCOAPODS: (.+)")
 	match := exp.FindStringSubmatch(content)
-	if match != nil && len(match) == 2 {
+	if len(match) == 2 {
```
Comment (godrei) [inline main.go]: nice!

File: main.go [hunk 2]
```diff
@@ -173,6 +134,13 @@ func cocoapodsVersionFromPodfileLock(podfileLockPth string) (string, error) {
 	return cocoapodsVersionFromPodfileLockContent(content), nil
 }
 
+func isCocoapodsInPath() bool {
+	if _, err := command.New("which", "pod").RunAndReturnTrimmedCombinedOutput(); err != nil {
```
Comment (godrei) [inline main.go]: NOTE:
```
_, err := command.New("which", "pod").RunAndReturnTrimmedCombinedOutput()
return err == nil
```

File: main.go [hunk 3]
```diff
@@ -196,19 +164,21 @@ func main() {
...
+		podVersionCmdSlice := []string{"pod", "--version"}
 
-	log.Donef("$ %s", command.PrintableCommandArgs(false, podVersionCmdSlice))
+		log.Donef("$ %s", command.PrintableCommandArgs(false, podVersionCmdSlice))
 
-	cmd, err := rubycommand.New("pod", "--version")
-	if err != nil {
-		failf("Failed to create command model, error: %s", err)
-	}
+		cmd, err := rubycommand.New("pod", "--version")
```
Comment (godrei) [inline main.go]: `log.Donef("$ %s", cmd.PrintableCommandArgs(false))` this way you can remove `podVersionCmdSlice := []string{"pod", "--version"}`

File: .ruby-version [hunk 4]
```diff
@@ -0,0 +1 @@
+2.5.3
```
Comment (godrei) [inline .ruby-version]: why did you lock ruby version?

File: main.go [hunk 5]
```diff
@@ -196,6 +157,7 @@ func main() {
 	fmt.Println()
 	log.Printf("System installed cocoapods version:")
 
+	// pod can be in the PATH as an rbenv shim and pod --version will return "rbenv: pod: command not found"
```
Comment (godrei) [inline main.go]: should not we install cocoapods before calling version?

Comment (lpusok) [reply main.go]: yes we should.

File: .ruby-version (hunk 4 above)
Comment (lpusok) [reply .ruby-version]: missed that.

**Pull request merged**